sandbox/src/test/java/org/robolectric/internal/bytecode/ClassInstrumentorTest.java

```diff
@@ -135,11 +167,21 @@ public void instrumentNativeMethod_generatesNativeBindingMethod() {
     String nativeMethodName = Shadow.directNativeMethodName("org.example.MyClass", "someFunction");
     MethodNode methodNode = findMethodNode(classNode, nativeMethodName);
 
+    assertThat(clazz.classNode.interfaces).contains(Type.getInternalName(ShadowedObject.class));
+    assertRoboDataField(clazz.getFields().get(0));
+
     assertThat(methodNode.access & Opcodes.ACC_NATIVE).isNotEqualTo(0);
     assertThat(methodNode.access & Opcodes.ACC_PRIVATE).isNotEqualTo(0);
     assertThat(methodNode.access & Opcodes.ACC_SYNTHETIC).isNotEqualTo(0);
   }
 
+  private static ClassNode createClassWithRegularNativeMethod() {
```

Comment:
nit: rename this method to `createClassWithRegularMethod`. The `createClassWithNativeMethod` below creates a method with the `native` keyword (i.e. used with JNI binding).